source/includes/databases-collections/databases-collections.php

```diff
@@ -0,0 +1,93 @@
+<?php
+require 'vendor/autoload.php';
+
+use MongoDB\BSON\Document;
+use MongoDB\Driver\ReadConcern;
+use MongoDB\Driver\ReadPreference;
+use MongoDB\Driver\WriteConcern;
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// Accesses the "test_database" database
+// start-access-database
+$db = $client->selectDatabase('test_database');
+// end-access-database
+
+// Accesses the "test_collection" collection
+// start-access-collection
+$collection = $client->test_database->selectCollection('test_collection');
```

Comment:
Since this file is using a mix of the shorthand syntax (e.g. `$client->test_database`) and explicit methods, would it be helpful to include notes in the RST that talks about how PHPLIB implements the [`__get() magic method`](https://www.php.net/manual/en/language.oop5.overloading.php#language.oop5.overloading.members) on Client and Database methods as an alternative way to access a child database or collection, respectively?

While it's implied from the examples on this page, you could also explicitly call out that the property access shorthand doesn't provide any way to specify options (in contrast to the explicit select methods).